src/pages/Events/Events.jsx

```diff
@@ -86,52 +86,55 @@ const Events = ({ themeMode }) => {
       />
 
       {isOpen && (
-        <Panel
-          themeMode={themeMode}
-          titleHeading="Your Events"
-          contentHeading="Events"
-          isOpen={isOpen}
-          setIsOpen={setIsOpen}
-        >
-          <Box className={`Events__button-panel`}>
-            <Button
-              variant="contained"
-              onClick={handleCreateEventClicked}
-              className={`Events__creat-event-button ${themeMode}`}
-            >
-              Create new event
-            </Button>
-
-            <Select
-              id="attendance-select"
-              className={`Events__attendance-select ${themeMode}`}
-              value={eventsFilterValue}
-              onChange={handleEventFilterClicked}
-            >
-              <MenuItem value={"Home"}>Home</MenuItem>
-              <MenuItem value={"Going"}>Going</MenuItem>
-              <MenuItem value={"Interested"}>Interested</MenuItem>
-              <MenuItem value={"Not Going"}>Not Going</MenuItem>
-            </Select>
-          </Box>
 
-          <Stack spacing={2} className={`Events__stack ${themeMode}`}>
-            {filteredEvents.map((event, i) => {
-              return (
-                <SingleEvent
-                  themeMode={themeMode}
-                  key={i}
-                  eventKey={i}
-                  eventData={event}
-                  events={events}
-                  setEvents={setEvents}
-                  setSnackbarOptions={setSnackbarOptions}
-                  setOpenSnackbar={setOpenSnackbar}
-                />
-              );
-            })}
-          </Stack>
-        </Panel>
+        <Box className={`Events ${themeMode}`} data-testid="events-component">
+          <Box className={`Events__header ${themeMode}`}>
+            <h3 className="Events__title">Your Events</h3>
+            <CloseButton handleClose={handleClose} />
```

Comment:
I ran this branch on my machine and it throws error that CloseButton is not defined so I fixed it by importing it. But now it throws error that handleClose is not defined.

Reply:
I saw this too but wasnt sure if it wss just omitted but I did notice a missing import.  I just figured it mightve gotten overlooked.

Reply:
Button is not a problem, handleClose function is a bigger case.

Reply:
Yep.  Overall the Event functions fine.  I wasnt sure if Manuel had just missed it somehow. 

It needs a little more work but is useable as is.  There is some nice-to-have functions I would like in it.

Reply:
I had to do a manual merge conflict and chose the code section of lines 90 to 93. I think it was because before I did't have this hardcoded Events panel and used the new reusable Panel instead, this might be way the handleClose function and and CloseButtons where not found, as they are imported only on the Panel component, if I remember correctly. 

Will check this and make a new commit, thanks for the feedback!

Reply:
This is why we write tests.  I saw the issue but was not sure how it came into existence so we can address it.

Reply:
Overall, I think Panel was a good idea.  Seeing recently we have people that want to contribute that may not know the overall layout and design.

Some docs to that effect mighr help as well.. but.. good catch.

Reply:
Ok, I've done some new commits, you can re-test now.

Reply:
> Overall, I think Panel was a good idea. Seeing recently we have people that want to contribute that may not know the overall layout and design.
> 
> Some docs to that effect mighr help as well.. but.. good catch.

We can definitely add some docs about the current available components.